modules/core/04-channel/types/msgs_test.go

```diff
@@ -494,9 +487,8 @@ func (suite *TypesTestSuite) TestMsgChannelUpgradeInitValidateBasic() {
 		suite.Run(tc.name, func() {
 			msg = types.NewMsgChannelUpgradeInit(
 				ibctesting.MockPort, ibctesting.FirstChannelID,
-				types.Channel{State: types.INITUPGRADE, Version: mock.Version},
-				clienttypes.NewHeight(0, 10000),
-				0,
+				types.NewUpgradeFields(types.UNORDERED, []string{ibctesting.FirstChannelID}, mock.Version),
```

Comment:
yes it should be, interesting, we are not validating a valid connection id format in validate basic. Is this something we should be doing?

Reply:
We could be. Only the first connection ID can be required to be in the "connection-X" format (thinking of multihop non len 1 context). I don't have a strong preference since it will fail on the handler trying to get the connection